source/educators/how-tos/rerun_course.rst

```diff
@@ -31,7 +31,3 @@ steps.
   while the process runs. The new course appears on your **My Courses**
   dashboard in Studio when configuration is complete.
 
-.. seealso::
```

Comment:
Why delete this See Also?

Reply:
Seems like there was some kind of uncertainty about how to format the targets. In the deleted See Also section I am seeing a few targets that look like they are the same, just with different formatting. I'll take note of this doc and any others and make sure they get See Also's.

Reply:
This was a stub file that ended up getting merged into course_rerun.rst, since we won't be using stub reusable files much. Resolved the issue, since course_rerun.rst has proper See Also section.